client/analysisui/gridcheckbox.js

```diff
@@ -14,6 +15,11 @@ const GridCheckbox = function(params) {
     this.$_subel = $('<label class="silky-option-checkbox silky-control-margin-' + this.getPropertyValue("margin") + '" style="white-space: nowrap;"></label>');
     this.$el = this.$_subel;
 
+    this.registerSimpleProperty("format", FormatDef.bool);
+
+    let horizontalAlign = this.getPropertyValue("horizontalAlignment");
+    this.$_subel.addClass("jmv-option-halign-" + horizontalAlign);
```

Comment:
i think you're better off using data attributes, rather than assembling class names, i.e.:

```
$el.attr('data-jmv-option-halign', horizontalAlign);
```

and then you can CSS select on that:

.class[data-jmv-option-halign='left']

or something like that.